lab-kyle/lib/read.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+
+function readAFile(files, results, callback) {
+  if (files.length === 0) {
+    callback(results);
+  } else {
+    fs.readFile(files[0], function(err, data) {
+      if (err) throw err;
+      files.shift();
+      let formatted = data.slice(0,8).toString('hex');
+      results.push(formatted);
+      console.log(formatted);
+      readAFile(files, results, callback);
+    });
+  }
+}
+
+function returnShit(results) {
```

Comment:
Define it here, export it, then pass it back in as a callback? Kind of a loopty-loop way to return the results.